src/components/applicantMain/ApplicantMain.tsx

```diff
@@ -25,9 +30,8 @@ import ResumeFolder from '../resumeFolder/ResumeFolder.tsx'
 import Support from '../support/Support.tsx'
 
 const drawerWidth = 240
-
-export default function ApplicantMain() {
-  const [page, setPage] = React.useState('')
+const ApplicantMain: React.FC = () => {
+  const [page, setPage] = useState<string>('')
```

Comment:
let's change the naming to be more descriptive for everyone

Reply:
changed to applicantPageType and employerPageType